### PR TITLE
Less charges for Sci flash

### PR DESCRIPTION
**Pull request by coolmankid12345**
## About the PR

Sci now prints two charge flashes instead of five charge flashes.
## Why / Balance

I've heard many complaints about Revs steamrolling because of high conversion rates because they literally only raid Sci and now I've lowered the charges in the flash because Revs should go other places, see the world man. While it may not fix the problem overall, I hope it'll help a little, and if not we are going to one charge. RIP plant-based flashes. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by UbaserB** [comment]
What do you think about providing a sprite for these "synthetic flashes"? Also, can you use these flashes to make borgs?

**Comment by coolmankid12345** [comment]
> What do you think about providing a sprite for these "synthetic flashes"? Also, can you use these flashes to make borgs?

If I knew how to sprite I would and yes you can.

**Comment by UbaserB** [comment]
> > What do you think about providing a sprite for these "synthetic flashes"? Also, can you use these flashes to make borgs?
> 
> If I knew how to sprite I would and yes you can.

if you want i can sprite it for you

**Comment by coolmankid12345** [comment]
> > > What do you think about providing a sprite for these "synthetic flashes"? Also, can you use these flashes to make borgs?
> > 
> > 
> > If I knew how to sprite I would and yes you can.
> 
> if you want i can sprite it for you

That would be greatly appreciated

**Comment by JoeHammad1844** [comment]
whats the difference between a regular and synthetic flash? aside from the name and flash amount, what makes them synthetic?

**Comment by deltanedas** [comment]
made by sci gamers and not sec bozos :trollface:

**Comment by JoeHammad1844** [comment]
do flashes grow on trees?

**Comment by coolmankid12345** [comment]
Anything is possible

**Comment by KingOnBudget** [comment]
This will make science even more of a target by headrevs while effecting the snowballing very slightly. Having 2 charges isn't an obstacle due to flashes themselves being very easy to make.

**Comment by crazybrain23** [comment]
"Synthetic" is the wrong word. It implies that the normal flashes are somehow organic.

**Comment by coolmankid12345** [comment]
> This will make science even more of a target by headrevs while effecting the snowballing very slightly. Having 2 charges isn't an obstacle due to flashes themselves being very easy to make.

We could always make it one charge

**Comment by coolmankid12345** [comment]
> "Synthetic" is the wrong word. It implies that the normal flashes are somehow organic.

Yeah I just stole the word from an idea someone said on Discord. I'm open to name suggestions

**Comment by JoeHammad1844** [comment]
i dont think that this should be be added, revs will just fill duffle bags with synthflashes and achieve the same thing, the only real change is more clutter from spent flashes and deconstructing the very station they are trying to rule to get more

**Comment by KompaniLinge** [comment]
Good PR, wouldn't say it solves the issue, though.
Wouldn't it make more sense to fix this issue by just making rev-flashes something that recharges/needs to be recharged/can be recharged so snowballing isn't as easy as it is now? This doesn't really fix the issue, its a band-aid on a giant gaping sword-wound. It helps, but not by a lot. I don't think this is really necessary, as whenever a new type of rev balancing is complete, this synthflash becomes useless.

**Comment by JoeHammad1844** [comment]
> Size increase too? You drive a hard bargain

NO PLEASE THATS NOT WHAT I MEAN

**Comment by JoeHammad1844** [comment]
i think this kindof undermines the entire point of revs, its a conversion antag and is SUPPOSED to be a snowball, you get a few followers at the beginning and then you do something ballsy to get some more flashes to convert more people and overpower sec with numbers. 

the answer isnt to nerf revs its to buff sec and give them better crow control gadgets like tear gas grenades or barrier grenades.

flashing people isnt without risk either because you could be outed as headrev and be hunted down, headrevs are still regular people and can be gunned down just as easily as anyone else

**Comment by coolmankid12345** [comment]
> Wasn’t the consensus that “borg eyes” should be a different item entirely (that can’t flash)? Given it’s not hard for Sci to print massive amounts of these, I don’t see limiting the number to 2 actually solving anything.

I didn't even know about borg eyes but it'd probably help in phasing out flashes from sci, I'll look into it if nobody else is.

**Comment by Nairodian** [comment]
> > "Synthetic" is the wrong word. It implies that the normal flashes are somehow organic.
> 
> Yeah I just stole the word from an idea someone said on Discord. I'm open to name suggestions

Don't worry about it. The name comes from SS13, it's a _synthetic_ flash designed to flash _synthetic_ creatures, it can still stun an organic but it was made for borgs. That's why it's called synthetic flash. (At least I'm assuming, this is actually me entirely bullshitting and just guessing based on what I've seen in my brief time in SS13 sci)

**Comment by Nairodian** [comment]
As for the issue at hand. I think this is a step in the right direction. It may not fix the problem out right but it'll improve the flow a bit more until a proper fix is made. This is already a good change just in an over all way, not even just for revos.

**Comment by EmoGarbage404** [comment]
i feel like having different names/designs is kinda unnecessary. Just add a variant that has reduced charges while keeping the same appearance. That's what 13 does, iirc.

**Comment by coolmankid12345** [comment]
> i feel like having different names/designs is kinda unnecessary. Just add a variant that has reduced charges while keeping the same appearance. That's what 13 does, iirc.

It's changed. I'll miss my flashes that grow on trees.

File: Resources/Prototypes/Entities/Objects/Weapons/security.yml [hunk 1]
```diff
@@ -138,6 +138,15 @@
       guides:
       - Security
 
+- type: entity
+  name: flash
+  parent: Flash
+  id: SciFlash
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Weapons/security.yml:144]: add a suffix for mappers

**Review by EmoGarbage404 — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/security.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Weapons/security.yml:144]: also no need to set name it inherits

**Pull request merged**